# *Conservation and loss of essential genes across the bacterial phylogeny*
## [Nashira H. Ridgeway](Nashira_Ridgeway_Resume_063025.pdf)
### *Presented at ISMB/ECCB 2025 in Liverpool, UK*

### [Code on GitHub](https://github.com/nashirag/)
[*Escherichia coli* Essential Gene Assembly](https://github.com/nashirag/Essential_Gene_Conservation/blob/main/E_coli_Essential_Gene_Compilation.ipynb)
[*Bacillus subtilis* Essential Gene Assembly](https://github.com/nashirag/Essential_Gene_Conservation/blob/main/B_subtilis_Essential_Gene_Compilation.ipynb)
[OrthoDB Ortholog Search](https://github.com/nashirag/Essential_Gene_Conservation/blob/main/OrthoDB_GeneFinding.ipynb)

### Related Links
- [Nashira's CV](Nashira_Ridgeway_Resume_063025.pdf)
- [Hospital Swabbing Metagenomics Project](CSM_Poster_June2025_PortraitFormat_outlinedForPrint.pdf)
- [Applying Machine Learning to Enzyme-Specific Post-translational Modification Prediction](CEEHRC_Nov2023_V3_outlined_27x40.pdf)
- [Nashira's GitHub](https://github.com/nashirag)
- [Nashira's LinkedIn](https://linkedin.com/in/NashiraRidgeway)

### References
**[1]** Baba, T.; Ara, T.; Hasegawa, M.; Takai, Y.; Okumura, Y.; Baba, M.; Datsenko, K. A.; Tomita, M.; Wanner, B. L.; Mori, H. Construction of Escherichia Coli K‐12 In‐frame, Single‐gene Knockout Mutants: The Keio Collection. Molecular Systems Biology 2006, 2 (1). https://doi.org/10.1038/msb4100050.

**[2]** Yamamoto, N.; Nakahigashi, K.; Nakamichi, T.; Yoshino, M.; Takai, Y.; Touda, Y.; Furubayashi, A.; Kinjyo, S.; Dose, H.; Hasegawa, M.; Datsenko, K. A.; Nakayashiki, T.; Tomita, M.; Wanner, B. L.; Mori, H. Update on the Keio Collection of Escherichia Coli Single‐gene Deletion Mutants. Molecular Systems Biology 2009, 5 (1). https://doi.org/10.1038/msb.2009.92.

**[3]** Goodall, E. C. A.; Robinson, A.; Johnston, I. G.; Jabbari, S.; Turner, K. A.; Cunningham, A. F.; Lund, P. A.; Cole, J. A.; Henderson, I. R. The Essential Genome of Escherichia Coli K-12. mBio 2018, 9 (1). https://doi.org/10.1128/mbio.02096-17.

**[4]** Rousset, F.; Cui, L.; Siouve, E.; Becavin, C.; Depardieu, F.; Bikard, D. Genome-Wide CRISPR-dCas9 Screens in E. Coli Identify Essential Genes and Phage Host Factors. PLoS Genet 2018, 14 (11), e1007749. https://doi.org/10.1371/journal.pgen.1007749.

**[5]** Warr, A. R.; Hubbard, T. P.; Munera, D.; Blondel, C. J.; Abel Zur Wiesch, P.; Abel, S.; Wang, X.; Davis, B. M.; Waldor, M. K. Transposon-Insertion Sequencing Screens Unveil Requirements for EHEC Growth and Intestinal Colonization. PLoS Pathog 2019, 15 (8), e1007652. https://doi.org/10.1371/journal.ppat.1007652.

**[6]** A. Ghomi, F.; Jung, J. J.; Langridge, G. C.; Cain, A. K.; Boinett, C. J.; Abd El Ghany, M.; Pickard, D. J.; Kingsley, R. A.; Thomson, N. R.; Parkhill, J.; Gardner, P. P.; Barquist, L. High-Throughput Transposon Mutagenesis in the Family Enterobacteriaceae Reveals Core Essential Genes and Rapid Turnover of Essentiality. mBio 2024, 15 (10). https://doi.org/10.1128/mbio.01798-24.

**[7]** Kobayashi, K.; Ehrlich, S. D.; Albertini, A.; Amati, G.; Andersen, K. K.; Arnaud, M.; Asai, K.; Ashikaga, S.; Aymerich, S.; Bessieres, P.; Boland, F.; Brignell, S. C.; Bron, S.; Bunai, K.; Chapuis, J.; Christiansen, L. C.; Danchin, A.; Débarbouillé, M.; Dervyn, E.; Deuerling, E.; Devine, K.; Devine, S. K.; Dreesen, O.; Errington, J.; Fillinger, S.; Foster, S. J.; Fujita, Y.; Galizzi, A.; Gardan, R.; Eschevins, C.; Fukushima, T.; Haga, K.; Harwood, C. R.; Hecker, M.; Hosoya, D.; Hullo, M. F.; Kakeshita, H.; Karamata, D.; Kasahara, Y.; Kawamura, F.; Koga, K.; Koski, P.; Kuwana, R.; Imamura, D.; Ishimaru, M.; Ishikawa, S.; Ishio, I.; Le Coq, D.; Masson, A.; Mauël, C.; Meima, R.; Mellado, R. P.; Moir, A.; Moriya, S.; Nagakawa, E.; Nanamiya, H.; Nakai, S.; Nygaard, P.; Ogura, M.; Ohanan, T.; O’Reilly, M.; O’Rourke, M.; Pragai, Z.; Pooley, H. M.; Rapoport, G.; Rawlins, J. P.; Rivas, L. A.; Rivolta, C.; Sadaie, A.; Sadaie, Y.; Sarvas, M.; Sato, T.; Saxild, H. H.; Scanlan, E.; Schumann, W.; Seegers, J. F. M. L.; Sekiguchi, J.; Sekowska, A.; Séror, S. J.; Simon, M.; Stragier, P.; Studer, R.; Takamatsu, H.; Tanaka, T.; Takeuchi, M.; Thomaides, H. B.; Vagner, V.; Van Dijl, J. M.; Watabe, K.; Wipat, A.; Yamamoto, H.; Yamamoto, M.; Yamamoto, Y.; Yamane, K.; Yata, K.; Yoshida, K.; Yoshikawa, H.; Zuber, U.; Ogasawara, N. Essential Bacillus Subtilis Genes. Proc. Natl. Acad. Sci. U.S.A. 2003, 100 (8), 4678–4683. https://doi.org/10.1073/pnas.0730515100.

**[8]** Peters, J. M.; Colavin, A.; Shi, H.; Czarny, T. L.; Larson, M. H.; Wong, S.; Hawkins, J. S.; Lu, C. H. S.; Koo, B.-M.; Marta, E.; Shiver, A. L.; Whitehead, E. H.; Weissman, J. S.; Brown, E. D.; Qi, L. S.; Huang, K. C.; Gross, C. A. A Comprehensive, CRISPR-Based Functional Analysis of Essential Genes in Bacteria. Cell 2016, 165 (6), 1493–1506. https://doi.org/10.1016/j.cell.2016.05.003.

**[9]** Koo, B.-M.; Kritikos, G.; Farelli, J. D.; Todor, H.; Tong, K.; Kimsey, H.; Wapinski, I.; Galardini, M.; Cabal, A.; Peters, J. M.; Hachmann, A.-B.; Rudner, D. Z.; Allen, K. N.; Typas, A.; Gross, C. A. Construction and Analysis of Two Genome-Scale Deletion Libraries for Bacillus Subtilis. Cell Systems 2017, 4 (3), 291-305.e7. https://doi.org/10.1016/j.cels.2016.12.013.

**[10]** Kuznetsov, D.; Tegenfeldt, F.; Manni, M.; Seppey, M.; Berkeley, M.; Kriventseva, E. V.; Zdobnov, E. M. OrthoDB V11: Annotation of Orthologs in the Widest Sampling of Organismal Diversity. Nucleic Acids Research 2023, 51 (D1), D445–D451. https://doi.org/10.1093/nar/gkac998.

